pkg/apis/v2beta2/receiver_types.go

```diff
@@ -244,6 +244,41 @@ type PushoverReceiver struct {
 	TmplText *ConfigmapKeySelector `json:"tmplText,omitempty"`
 }
 
+// FeishuChatBot is the configuration of ChatBot
+type FeishuChatBot struct {
+	// The webhook of ChatBot which the message will send to.
```

Comment:
// The webhook of ChatBot which the message will send to.
=>
	// The webhook of the ChatBot to which the message will be sent.